### ISSUE TITLE
Support for Streaming in the Assistants API

### ISSUE THREAD

**Issue by sevzas** [JZO001/Forge.OpenAI#12]
This is a feature request.

OpenAI API offers streaming notification for changes to message and run objects.

https://platform.openai.com/docs/api-reference/assistants-streaming

It would be great in Forge.OpenAI supported this feature.

**Comment by JZO001** [comment]
Yes, this is definetly the next dev task on my side. I will notify, when its ready.
